js/bitget.js

```diff
@@ -1840,7 +1840,7 @@ module.exports = class bitget extends Exchange {
             fee = {
                 'code': currencyCode, // kept here for backward-compatibility, but will be removed soon
                 'currency': currencyCode,
-                'cost': feeAmount,
+                'cost': Precise.stringNeg (feeAmount),
```

Comment:
Would it make sense to use the absolute value?
```suggestion
                'cost': Precise.stringAbs (feeAmount),
```

Reply:
I put negative in case of rebate. But I don't know if it possible to have rebate on Bitget, I don't found any offers. We can try to use absolute.

Reply:
I think it's ok to use `stringNeg` here